authentication.js

```diff
@@ -22,11 +22,34 @@ const testAuth = (z /*, bundle*/) => {
 
 module.exports = {
     type: "custom",
+
     // Define any auth fields your app requires here. The user will be prompted to enter this info
     // when they connect their account.
     fields: [
-        {key: "apiKey", label: "API Key", required: true, type: "string"}
+        {
+            key: "apiKey",
+            label: "API Key",
+            required: true,
+            type: "string",
+        },
+        {
+            key: "server",
+            label: "Server",
+            type: "string",
+            helpText: "Handshake server to use, e.g. app.handshake.com, sandbox.handshake.com etc",
+            default: "app.handshake.com",
+            required: true,
+        },
+        {
+            key: "include_temp_auth",
+            label: "Include temp API key",
+            type: "boolean",
+            default: "no",
+            helpText: "Includes a temporary API key in each webhook. Helps avoid " +
+                "hardcoding your API key in your zap if you plan to make manual API calls.",
+        },
```

Comment:
Users can opt to include temp API auth in their hooks, especially useful if you plan to do any manual webhook or code stuff in your zap.